packages/webapp/src/components/List/ListItems/IconDescriptionCheckbox/styles.module.scss

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+.listItem {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid var(--green100);
+  box-shadow: 0 2px 10px #ededed;
+  border-radius: 4px;
+  width: 100%;
+  min-height: 64px;
+  padding: 0 4px;
+  overflow: hidden;
+  cursor: pointer;
+}
+
+.icon {
+  width: 36px;
+  height: 36px;
+  margin: 18px 9px;
```

Comment:
Super minor thing, but Loïc has mentioned that he tries to maintain an 8px grid in his designs. I think in this case the spacing is a bit weird because the icons are different sizes, but when in doubt I'd suggest sticking to multiples of 8, so in this case 16 and 4 or 16 and 8.

Reply:
Yeah good to know -- I saw 9 and 18 in figma. I can fix that up.